drools-ansible-rulebook-integration-main/src/test/java/org/drools/ansible/rulebook/integration/main/SlownessTest.java

```diff
@@ -0,0 +1,26 @@
+package org.drools.ansible.rulebook.integration.main;
+
+import org.drools.ansible.rulebook.integration.api.rulesengine.AbstractRulesEvaluator;
+import org.drools.ansible.rulebook.integration.api.rulesengine.SlowAutomaticPseudoClock;
+import org.junit.Test;
+
+public class SlownessTest {
+
+    @Test
+    public void testOnceAfter() {
+        try {
+            System.setProperty(AbstractRulesEvaluator.AUTOMATIC_PSEUDO_CLOCK_CUSTOM_CLASS_NAME_PROPERTY, SlowAutomaticPseudoClock.class.getName());
+            System.setProperty(SlowAutomaticPseudoClock.DELAY_PROPERTY, "11000");
+            System.setProperty(SlowAutomaticPseudoClock.AMOUNT_PROPERTY, "5000");
+            Main.execute("56_once_after.json");
+
+            // No assertions, just check a warning. TODO : modify Main so that we can assert matches
+            // WARN org.drools.ansible.rulebook.integration.api.rulesengine.SessionStatsCollector - r1 is fired with a delay of 4013 ms
```

Comment:
Currently we can just confirm the WARN log. Maybe I can modify `Main` so that we can assert matches.